pkg/gateway/multiparts/db_tracker.go

```diff
@@ -92,3 +103,39 @@ func (m *dbTracker) Delete(ctx context.Context, uploadID string) error {
 	})
 	return err
 }
+
+func Migrate(d db.Database, writer io.Writer) error {
+	ctx := context.Background()
+	je := json.NewEncoder(writer)
+
+	// Create header
+	err := je.Encode(export.Header{
+		Version:   kv.MigrateVersion + 1,
+		Timestamp: time.Now(),
+	})
+	if err != nil {
+		return err
+	}
+
+	entries := make([]MultipartUpload, 0)
+	err = d.Select(ctx, &entries, "SELECT * FROM gateway_multiparts")
+	if err != nil {
+		return err
+	}
+	for i := range entries {
+		pr := protoFromMultipart(&entries[i]) // gosec error WA
+		value, err := proto.Marshal(pr)
+		key := []byte(kv.FormatPath(multipartPrefix, entries[i].UploadID))
+		if err != nil {
+			return err
+		}
+		err = je.Encode(kv.Entry{
```

Comment:
The Go's json encoder produce a line based file? or just serialize all the message one by one?

Reply:
https://pkg.go.dev/encoding/json#Encoder.Encode

> ...Encode writes the JSON encoding of v to the stream, followed by a newline character....